browser/util/generateReport.ts

```diff
@@ -0,0 +1,105 @@
+/* Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+const months = [
+  undefined,
```

Comment:
Why have an undefined option when you can default to Jan?

Reply:
I had undefined just to fill the place for month[0] so I can start January at month[1] just as an easy way for me to convert the numeric month to words. I can set month[0] as January if you would like.